dune-project

```diff
@@ -0,0 +1,21 @@
+(lang dune 3.10)
+
+(name diff)
+
+(source (github skolemlabs/diff))
+
+(generate_opam_files true)
+
+(version 1.0.0)
+
+(authors "Zach Baylin <zach.baylin@skolem.com>")
+(maintainers "Zach Baylin <zach.baylin@skolem.com>")
+
+(package
+ (name diff)
+ (synopsis "A library for generating diffs from product types.")
+ (depends
```

Comment:
**Suggestion:** I suppose the intention of this PR might not be to pass all of the `dune-release lint` criteria, but I noticed the license field is missing here (and perhaps some other related info) -- do you want to add that stuff in this PR?

Reply:
Oh good idea